app/models/concerns/waste_carriers_engine/can_change_registration_status.rb

```diff
@@ -78,82 +83,34 @@ def log_status_change
     # Guards
     def renewal_allowed?
       return true if renewal_application_submitted?
-      return true if in_expiry_grace_window?
+      return true if renewal_declaration_confirmed?
 
       # The only time an expired registration can be renewed is if the application has previously been submitted,
```

Comment:
Think this comment needs updating, as it can now be renewed once the declaration is accepted, not just if it's been submitted.